sql/plan/show_triggers.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2020 Liquidata, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plan
+
+import (
+	"github.com/dolthub/go-mysql-server/sql"
+	"strings"
+	"time"
+)
+
+type ShowTriggers struct {
+	db       sql.Database
+	Triggers []*CreateTrigger
+}
+
+var _ sql.Databaser = (*ShowTriggers)(nil)
+var _ sql.Node = (*ShowTriggers)(nil)
+
+// NewShowCreateTrigger creates a new ShowCreateTrigger node for SHOW TRIGGER statements.
+func NewShowTriggers(db sql.Database) *ShowTriggers {
+	return &ShowTriggers{
+		db: db,
+	}
+}
+
+// String implements the sql.Node interface.
+func (s *ShowTriggers) String() string {
+	return "SHOW TRIGGERS"
+}
+
+// Resolved implements the sql.Node interface.
+func (s *ShowTriggers) Resolved() bool {
+	_, ok := s.db.(sql.UnresolvedDatabase)
+	return !ok
+}
+
+// Children implements the sql.Node interface.
+func (s *ShowTriggers) Children() []sql.Node {
+	return nil
+}
+
+// Schema implements the sql.Node interface.
+func (s *ShowTriggers) Schema() sql.Schema {
+	return sql.Schema{
```

Comment:
Probably better as a var

Reply:
I actually went this route as it looks like most of those that don't return nil return their schema this same way, but I'll change it.